pages/portfolio.html

```diff
@@ -4,9 +4,70 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="/styles/style.css" rel="stylesheet">
 </head>
-<body>
-  
+<body class="grid-portfolio">
+  <header>
+    <h1>My portfolio</h1>
+    <nav>
+      <ul>
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="portfolio.html">Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section>
+        <div>
```

Comment:
I notice this div doesn't have a class on it. Do you use it for styling? If it's not being used you can get rid of this div tag since you have the main tag and section tag helping to provide structure to the page